src/components/dialog/dialog.svelte

```diff
@@ -23,10 +24,11 @@
   export let modal = true
   export let showClose = false
   export let showBack = false
-  export let size: DialogSizes = 'normal'
+  export let size: DialogSizes = undefined
   export let escapeCloses = true
   export let backdropClickCloses = true
   export let animate = true
+  export let mobileBreakpoint = 375
```

Comment:
Should this be specified via css?

Reply:
The idea was to allow consumers to determine the breakpoint so that it's not locked in.